elytron/src/test/java/org/wildfly/extension/elytron/CredentialStoreTestCase.java

```diff
@@ -221,13 +222,14 @@ public void testExportPublicKey() {
         Assert.assertTrue(publicKey.contains("ssh-rsa AAAAB3NzaC1yc2EAAAADAQABAAABAQ"));
     }
 
+    @Ignore
```

Comment:
@lvydra Feel free to let us know when this is ready for another review. Since this test is currently being ignored, it looks like you're working on fixing the test failures, right?

Reply:
Hi @fjuma, yes, I'm working on fixing test failures. Ignored tests are written to test `setAlternatives` behavior, which is currently not working as it's supposed to, so I'm looking for a way how to fix that.